devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -237,3 +239,15 @@ def configure() {
 	// battery minReport 30 seconds, maxReportTime 6 hrs by default
 	return refresh() + zigbee.batteryConfig() + zigbee.temperatureConfig(30, 300) // send refresh cmds as part of config
 }
+
+private isFirmwareVersionLessThan(major, minor, build) {
+	def deviceFwVer = "${device.getFirmwareVersion()}"
```

Comment:
Actually just realized that it is possible for this to be `null`.  There can be times when we don't know about the devices firmware.  I think what we want to do here is if the value is `null` return `true` but then the naming of the function is kind of weird.  We could rename it something like `useOldBatteryReporting` instead

Reply:
Ah right! Thanks for catching that null case. I had included that when I was designing it in my head.

As for the function naming.... hmmm...